### ISSUE TITLE
Power Point 2013-Info about the thumbnails is not given as given in previous versions of Power Point

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3968]
**Reported by sumandogra on 2014-03-11 12:48**
Power Point 2013
On reaching the thumbnails with F6, NVDA does not inform about the thumbnails pane IT INFORMS ABOUT the slide number and the total number of slides.
Steps to reproduce:
1.  Open a presentation in Power Point 2013.
2.  Press F6 to navigate in various panes.
3.  On reaching the thumbnails, no info about the pane is given.  NVDA between announces the slide number.
4.  In Power point 2010, the info about the thumbnails is given.
5.

**Comment by leonardder** [comment]
Is there someone with Powerpoint 2013 who could try to reproduce this? Would be greatly appreciated.

**Comment by Adriani90** [comment]
@sumandogra am I understanding this issue correctly? You want NVDA to report number of slide and total nomber of slides along with "thumbnail". Right? Whic information should be reported by NVDA exactly?
I am testing in Powerpoint 2016. Thumbnail is reported and the slide number as well.

**Comment by Adriani90** [comment]
This is not reproducible in Office 2026 and Office 2013 has reached end of life many years ago. We didn't get any further updates from the initial author. I am closing as abandoned.
